docs/source/user_guide/esb_3857.csv

```diff
@@ -0,0 +1,3 @@
+element_type,osmid,nodes,addr:city,addr:housenumber,addr:postcode,addr:state,addr:street,building,building:colour,building:levels,building:use,check_date,construction_date,ele,gnis:feature_id,height,name,name:en,name:es,name:he,name:hi,name:ko,name:ru,name:uk,name:zh,nycdoitt:bin,office,opening_hours,phone,start_date,tourism,website,wheelchair,wikidata,wikipedia,geometry
```

Comment:
It looks like the relevant data here is the polygon shape, but this has names in 9 languages and other metadata. Can we strip this to only the relevant columns? Maybe geometry and place name only?